config/config.go

```diff
@@ -15,23 +15,32 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
-	KafkaAddr                  []string      `envconfig:"KAFKA_ADDR"`
-	KafkaVersion               string        `envconfig:"KAFKA_VERSION"`
-	KafkaOffsetOldest          bool          `envconfig:"KAFKA_OFFSET_OLDEST"`
-	KafkaNumWorkers            int           `envconfig:"KAFKA_NUM_WORKERS"`
-	KafkaSecProtocol           string        `envconfig:"KAFKA_SEC_PROTO"`
-	KafkaSecCACerts            string        `envconfig:"KAFKA_SEC_CA_CERTS"`
-	KafkaSecClientCert         string        `envconfig:"KAFKA_SEC_CLIENT_CERT"`
-	KafkaSecClientKey          string        `envconfig:"KAFKA_SEC_CLIENT_KEY"          json:"-"`
-	KafkaSecSkipVerify         bool          `envconfig:"KAFKA_SEC_SKIP_VERIFY"`
-	PublishedContentGroup      string        `envconfig:"KAFKA_PUBLISHED_CONTENT_GROUP"`
-	PublishedContentTopic      string        `envconfig:"KAFKA_PUBLISHED_CONTENT_TOPIC"`
 	BatchSize                  int           `envconfig:"BATCH_SIZE"`
 	BatchWaitTime              time.Duration `envconfig:"BATCH_WAIT_TIME"`
 	ElasticSearchAPIURL        string        `envconfig:"ELASTIC_SEARCH_URL"`
 	AwsRegion                  string        `envconfig:"AWS_REGION"`
 	AwsService                 string        `envconfig:"AWS_SERVICE"`
 	SignElasticsearchRequests  bool          `envconfig:"SIGN_ELASTICSEARCH_REQUESTS"`
+	StopConsumingOnUnhealthy   bool          `envconfig:"STOP_CONSUMING_ON_UNHEALTHY"`
+	Kafka                      *Kafka
+}
+
+// Kafka contains the config required to connect to Kafka
+type Kafka struct {
+	PublishedContentGroup     string   `envconfig:"KAFKA_PUBLISHED_CONTENT_GROUP"`
```

Comment:
Is it worth ordering these alphabetically?

Reply:
Ok, I will do that.